parquet-tools/src/main/java/parquet/tools/command/CatCommand.java

```diff
@@ -52,7 +69,11 @@ public void execute(CommandLine options) throws Exception {
       PrintWriter writer = new PrintWriter(Main.out, true);
       reader = new ParquetReader<SimpleRecord>(new Path(input), new SimpleReadSupport());
       for (SimpleRecord value = reader.read(); value != null; value = reader.read()) {
-        value.prettyPrint(writer);
+        if (options.hasOption('j')) {
+          value.prettyPrintJson(writer);
```

Comment:
At your preference. This sounds good. Thanks again for contributing. The main thing I care about is the isArray and isMap decision to be driven by the schema more explicitly. 

The second approach could be as simple as creating subclasses of SimpleRecordConverter for Map and List that override start() to create a new Simple{List,Map}Record() with toJsonObject overridden to do the right thing. They would be used there:
https://github.com/apache/incubator-parquet-mr/blob/a0c77b6a442e2c4a355a4b145898bed976f23bb4/parquet-tools/src/main/java/parquet/tools/read/SimpleRecordConverter.java#L64